[2025-08-09T19:48:55.018+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-09T19:48:55.053+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: xrs_update_12h.update_swpc_recent scheduled__2025-08-09T00:00:00+00:00 [queued]>
[2025-08-09T19:48:55.062+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: xrs_update_12h.update_swpc_recent scheduled__2025-08-09T00:00:00+00:00 [queued]>
[2025-08-09T19:48:55.071+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2025-08-09T19:48:55.085+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): update_swpc_recent> on 2025-08-09 00:00:00+00:00
[2025-08-09T19:48:55.095+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'xrs_update_12h', 'update_swpc_recent', 'scheduled__2025-08-09T00:00:00+00:00', '--job-id', '64', '--raw', '--subdir', 'DAGS_FOLDER/xrs_update_daily.py', '--cfg-path', '/tmp/tmpuu5snejw']
[2025-08-09T19:48:55.096+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61 DeprecationWarning: This process (pid=497) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-08-09T19:48:55.098+0000] {standard_task_runner.py:91} INFO - Job 64: Subtask update_swpc_recent
[2025-08-09T19:48:55.098+0000] {standard_task_runner.py:63} INFO - Started process 498 to run task
[2025-08-09T19:48:55.164+0000] {task_command.py:426} INFO - Running <TaskInstance: xrs_update_12h.update_swpc_recent scheduled__2025-08-09T00:00:00+00:00 [running]> on host 9d4e07229e62
[2025-08-09T19:48:55.288+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='you' AIRFLOW_CTX_DAG_ID='xrs_update_12h' AIRFLOW_CTX_TASK_ID='update_swpc_recent' AIRFLOW_CTX_EXECUTION_DATE='2025-08-09T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-09T00:00:00+00:00'
[2025-08-09T19:48:55.289+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-09T19:48:55.304+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-09T19:48:55.306+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /opt/project/src/data/xrs_update.py --mode update --since-minutes 20160 --keep-days 180']
[2025-08-09T19:48:55.323+0000] {subprocess.py:86} INFO - Output:
[2025-08-09T19:48:57.143+0000] {subprocess.py:93} INFO - [INFO] retention applied: >= 2025-02-10 19:48:57.126904+00:00
[2025-08-09T19:48:57.143+0000] {subprocess.py:93} INFO - {"merged_rows_total": 10077, "range": ["2025-08-02 19:49:00+00:00", "2025-08-09 19:45:00+00:00"]}
[2025-08-09T19:48:57.234+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-09T19:48:57.235+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-09T19:48:57.272+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=xrs_update_12h, task_id=update_swpc_recent, run_id=scheduled__2025-08-09T00:00:00+00:00, execution_date=20250809T000000, start_date=20250809T194855, end_date=20250809T194857
[2025-08-09T19:48:57.321+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-09T19:48:57.340+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-09T19:48:57.350+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
